Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -129,15 +129,30 @@ init python:
         #TODO: Mac vers (if possible)
         return store.mas_windowreacts.can_show_notifs and mas_getActiveWindow(True) == config.name
 
-    def mas_isInActiveWindow(keywords):
+    def mas_isInActiveWindow(keywords, non_inclusive=False):
         """
         Checks if ALL keywords are in the active window name
 
         IN:
-            List of keywords
+            keywords:
+                List of keywords to check for
+
+            non_inclusive:
+                Whether or the not the list is checked non-inclusively
+                (Default: False)
         """
+
+        #Don't do work if we don't have to
+        if not store.mas_windowreacts.can_show_notifs:
+            return False
+
+        #Otherwise, let's get the active window
         active_window = mas_getActiveWindow()
-        return store.mas_windowreacts.can_show_notifs and len([s for s in keywords if s.lower() not in active_window]) == 0
+
+        if not non_inclusive:
+            return len([s for s in keywords if s.lower() not in active_window]) == 0
+        else:
+            return len([s for s in keywords if s.lower() in active_window]) > 0
```

Comment:
use set intersection for better comparing of inclusive/exclusive checking. (See `_filterEvent` in definitions)